packages/application/src/shell.ts

```diff
@@ -789,6 +792,34 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     }
   }
 
+  /**
+   * Activate specified Area widget.
+   */
```

Comment:
I think the following comment will do the trick. @fcollonval, does this look good to you?

```suggestion
  /**
   * Activate specified Area widget.
   * 
   * ### Notes
   * The alpha version of this method only supports activating the "main" area.
   *
   * @alpha 
   * @param area Name of area to activate
   */
```